docs/programming-guide.md

```diff
@@ -1099,6 +1099,34 @@ for details.
 </tr>
 </table>
 
+#### Asynchronous Actions
+Spark provide asynchronous actions to execute two or more actions concurrently, these actions are execute asynchronously without blocking each other. 
```

Comment:
"without blocking each other": actions never block each other anyway. They don't block the calling thread.